docs/docs/workflows/troubleshooting.md

```diff
@@ -54,6 +54,8 @@ When in doubt, check if the encountered [issue is known](https://github.com/edge
 
 :::
 
+If you want to restore the old attestation config after a failed upgrade, you can read the values in the config map `join-config-backup`. On each attestation config upgrade, the current values are stored there.
```

Comment:
This needs more context. I suggest
* extending the first paragraph of this section that a failed upgrade can also be a reason for this error
* making this paragraph a `:::tip` admonition and rearrange the sentences like: "on upgrade, a backup of the join config is created ... you can get the values ... put them in your configuration file"